Add include line for the DeePMD-kit plugin
See DeepMD documentation: https://docs.deepmodeling.com/projects/deepmd/en/stable/install/install-lammps.html#with-cmake
Author: Lara Peeters (UGent)
diff -ruN lammps-stable_29Aug2024.orig/cmake/CMakeLists.txt lammps-stable_29Aug2024/cmake/CMakeLists.txt
--- lammps-stable_29Aug2024.orig/cmake/CMakeLists.txt	2024-08-30 02:47:52.000000000 +0200
+++ lammps-stable_29Aug2024/cmake/CMakeLists.txt	2025-06-20 10:17:41.184817337 +0200
@@ -1091,3 +1091,5 @@
     message(STATUS "Collecting code coverage data")
   endif()
 endif()
+
+include(${deepmd_source_dir}/source/lmp/builtin.cmake)
